### ISSUE TITLE
SRGAN - Adversarial Loss function

### ISSUE THREAD

**Issue by Sumeet2807** [eriklindernoren/PyTorch-GAN#176]
I see the loss implemented here are as follows : -

# Losses
criterion_GAN = torch.nn.MSELoss()
criterion_content = torch.nn.L1Loss()

It is not same as the original paper, right ? Adversarial loss should be a cross entropy loss if I am not wrong. Does this setup perform better than the original paper formulation ?
